server/routes/users/index.ts

```diff
@@ -36,27 +37,63 @@ const upload = multer({
 const app = express();
 
 // sign-up user
-app.post('/api/v1/users', celebrate(userSchema), wrap(userController.postUsers));
+app.post(
+	'/api/v1/users',
+	celebrate(userSchema),
+	wrap(userController.postUsers)
+);
 
 // log-in the user and return their 2 JWT tokens
 app.post('/api/v1/login', verifyUser, wrap(userController.login));
 
 // return the access token providing that the refresh token is valid
-app.get('/api/v1/token', validateRefreshJWT, wrap(userController.getAccessToken));
+app.get(
+	'/api/v1/token',
+	validateRefreshJWT,
+	wrap(userController.getAccessToken)
+);
 
 // logout user
 app.delete('/api/v1/logout', validateRefreshJWT, wrap(userController.logout));
 
 // get user reactions
-app.get('/api/v1/users/:username/reactions', validateUsernameParam, wrap(userController.getUserReactions));
+app.get(
+	'/api/v1/users/:username/reactions',
+	validateUsernameParam,
+	wrap(userController.getUserReactions)
+);
 
 // get user data such as reputation and number of posts
-app.get('/api/v1/users/:username', validateUsernameParam, wrap(userController.getUserData));
+app.get(
+	'/api/v1/users/:username',
+	validateUsernameParam,
+	wrap(userController.getUserData)
+);
 
 // change the user image
-app.post('/api/v1/users/:username/icon', validateRefreshJWT, validateUsernameCookie, upload.single('image'), wrap(userController.changeUserIcon));
+app.post(
+	'/api/v1/users/:username/icon',
+	validateRefreshJWT,
+	validateUsernameCookie,
+	upload.single('image'),
+	wrap(userController.changeUserIcon)
+);
 
 // get the icon of a user
-app.get('/api/v1/users/:username/icon', validateUsernameParam, wrap(userController.getUserIcon));
+app.get(
+	'/api/v1/users/:username/icon',
+	validateUsernameParam,
+	wrap(userController.getUserIcon)
+);
+
+// set up notifications
+// if the server runs on a different address and create-react-app is proxying SSE won't work so have to send request directly to server
+app.get(
+	'/api/v1/users/:username/notifications',
+	cors({ credentials: true, origin: 'http://localhost:3000' }),
```

Comment:
This is a good setup. But maybe you'd like to use env vars to set the origin? And is the app deployed and tested in different environments?

Reply:
I tested notifications while having the express server serve the built react app and notifications seem to work still.